scripts/doctor.sh

```
@@ -0,0 +1,583 @@
+fi
+
+# Determine package manager
+if [[ "$OS" == "macos" ]]; then
+    PKG_MANAGER="brew"
+elif [[ "$OS" == "linux" ]]; then
+    if command -v apt-get &> /dev/null; then
+        PKG_MANAGER="apt"
+    elif command -v yum &> /dev/null; then
+        PKG_MANAGER="yum"
+    else
+        PKG_MANAGER="unknown"
```

Shellcheck (0.11.0)

[warning] PKG_MANAGER appears unused. Verify use (or export if used externally).

(SC2034)